src/Components/Layout/NavBar.js

```diff
@@ -19,43 +20,74 @@ class NavBar extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      Navlink: ["Home", "Features", "Price", "Design"]
+      Navlink: [
+        {
+          name: "Home",
+          link: "/",
+        },
+        {
+          name: "Pricing",
+          link: "/pricing",
+        },
+      ],
     };
   }
 
   render() {
     // maps the links to be displayed at the top, eventually one of them can be like "Hello " + this.props.studentUsername
-    let navlinks = this.state.Navlink.map(item => <Nav.Link>{item}</Nav.Link>);
+    let navlinks = this.state.Navlink.map((item) => (
+      <Nav.Link>
+        <Link className="whiteLink" to={item.link}>
+          {item.name}
+        </Link>
+      </Nav.Link>
+    ));
     return (
       <div>
         <Navbar bg="dark" variant="dark" expand="md">
           <Navbar.Brand>TrackIT</Navbar.Brand>
           <Navbar.Toggle aria-controls="basic-navbar-nav" />
           <Navbar.Collapse id="basic-navbar-nav">
-            <Nav className="mr-auto">{navlinks}</Nav>
+            <Nav className="mr-auto">
```

Comment:
href goes in Nav.Link
Also instead of hard coding this, would it be also possible to save this in the state and have it get mapped out the same way other links get mapped out? Or because it redirects to another site instead of a component is that more tricky?

Reply:
> Also instead of hard coding this, would it be also possible to save this in the state and have it get mapped out the same way other links get mapped out? Or because it redirects to another site instead of a component is that more tricky?

I thought of different ways on how I could do that to keep the code simple and consistent. However, the `<Link>` tag only redirects to other endpoints on our `router`. So just to make the code consistent, I would need to have a `Docs.js` component in the `Components/Pages/` directory that servers the `README` file from githiub.

I figured its not worth that much of a hassle to go around the hardcode.

Reply:
Gotcha, makes sense, we can think about that as a post-mvp goal